packages/babel-cli/test/fixtures/babel-external-helpers/--whitelist/stdout.txt

```diff
@@ -14,6 +14,9 @@
   function _createClass(Constructor, protoProps, staticProps) {
     if (protoProps) _defineProperties(Constructor.prototype, protoProps);
     if (staticProps) _defineProperties(Constructor, staticProps);
+    Object.defineProperty(Constructor, "prototype", {
+      writable: false
```

Comment:
The `writable` defaults to `false` when a property is defined by `Object.defineProperty`. Since helper is shipped in user code. We can just call
`Object.defineProperty(Constructor, "prototype", {})`.

Reply:
Given that this function can be shimmed, and writability might not be supported (in es5-sham, it throws), this is a good change.

Reply:
It defaults to `false` only if the property does not exist yet:
```js
function F() {}
Object.defineProperty(F, "prototype", {});
Object.getOwnPropertyDescriptor(F, "prototype").writable; // true
```